features/step_definitions/logging.rb

```diff
@@ -0,0 +1,128 @@
+### logging related step definitions
+# for 4.x we default logging installation is via OLM only
+
+Given /^logging service has been installed successfully$/ do
```

Comment:
can we remove old code related to these two steps
```
there should be (\d+) (logging|metrics) services? installed
all deployer logging pods are running in the#{OPT_QUOTED} project
```

Reply:
For 4.x I think the file `features/step_definitions/logging_metrics.rb` can be removed.

Reply:
yes, please remove dead code

Reply:
@pruan-rht   It is better to split this step to cluster-logging operator ready and provide CRD xxx are provided.
Most of cases are as following
1.  Cluster logging operator is ready.
2. Provide different type of resource file or modify the current resource file
So it is very importance to have a Cluster logging operator ready step.

Reply:
I don't see old file removed. Maybe missed to commit?

Reply:
There are some methods in there that might still be applicable.  Since there is currently no conflict, I would prefer to have the removal of the file be done in a separate PR>

Reply:
ok, but remember to do it ;)